contracts/SlashIndicator.sol

```diff
@@ -25,6 +30,13 @@ contract SlashIndicator is ISlashIndicator,System,IParamSubscriber, IApplication
   uint256 public  misdemeanorThreshold;
   uint256 public  felonyThreshold;
 
+  // BEP-126 Fast Finality
+  uint256 public constant FINALITY_DISTANCE = 11;
+  uint256 public constant FINALITY_SLASH_REWARD_RATIO = 20;
```

Comment:
`INIT_FINALITY_DISTANCE` and `INIT_FINALITY_SLASH_REWARD_RATIO` is better

Reply:
Fixed